packages/react-core/src/helpers/OUIA/OUIA.md

```diff
@@ -0,0 +1,93 @@
+---
+id: Open UI Automation
+section: developer-resources
+---
+
+## About Open UI Automation
+
+Open UI Automation (OUIA) is a specification of certain key guidelines to follow when creating a new
+web framework or application. The goal is to ease the burden of individuals wishing to create and
+maintain automated testing environments. OUIA standardizes some HTML attributes and web page
+behavior. It helps to create trivial and unambiguous locators of the elements in the DOM for
+various automated test suites.
+
+### Why not to use `id`, `ARIA` etc.?
```

Comment:
```suggestion
### Why not to use `id`, `ARIA` etc.?
``` 
Can we either replace or remove "etc." ? It's not always easily understood by global audiences, especially for folks whose native language isn't rooted in Latin. 

Also, should "Why not to use" be "Why not use" ?